src/Sulu/Bundle/SecurityBundle/Controller/ResettingController.php

```diff
@@ -228,7 +240,9 @@ public function sendEmailAction(Request $request)
             $email = $this->getEmail($user);
             $this->sendTokenEmail($user, $this->getSenderAddress($request), $email, $token);
         } catch (\Exception $ex) {
-            // do nothing
+            if ($this->logger instanceof LoggerInterface) {
```

Comment:
This check can be avoided when fallback on __construct to `new NullLogger`.

Reply:
Done